infrastructure/cdn-in-a-box/README.md

```diff
@@ -1,44 +1,141 @@
-`traffic_ops/traffic_ops.rpm`, `traffic_portal/traffic_portal.rpm`.
-    
-    cd infrastructure/cdn-in-a-box
-    docker-compose up --build
+> Note: You will likely see a warning about an untrusted or invalid certificate for
+> components that serve over HTTPS (Traffic Ops & Traffic Portal). If you
+> are sure that you are looking at `https://localhost:N` for some integer `N`, these
+> warnings may be safely ignored via the e.g. `Add Exception` button (possibly hidden
+> behind e.g. `Advanced Options`).
+
+### Traffic Ops
+The API and legacy UI for the CDN
+* URLs:
+	* New Golang endpoints: [`https://localhost:6443`](https://localhost:6443)
+	* Limited, Legacy Perl endpoints: [`https://localhost:60443`](https://localhost:60443)
+* Login Credentials:
+	* Username: `admin`
+	* Password: `twelve`
+
+### Traffic Portal
+The modern UI for the CDN
+* URL: [`https://localhost`](https://localhost)
+* Login Credentials:
+	* Username: `admin`
+	* Password: `twelve`
+
+### Traffic Ops PostgreSQL Database
+This holds the configuration information for the entire CDN. It is normally accessed
+directly only by Traffic Ops.
+* URL: [`postgres://traffic_ops:twelve@localhost:5432/traffic_ops`](postgres://traffic_ops:twelve@localhost:5432/traffic_ops)
+* Login Credentials:
+	* Username: `traffic_ops`
+	* Password: `twelve`
+* Port: 5432
+* Database: `traffic_ops`
+
+### Traffic Vault
+A secure storage server for private keys used by Traffic Ops
+* Port: 8010
+
+### Edge-Tier Cache
+An edge-tier cache sits at the outermost extremity of the CDN, typically serving content
+directly to the user from either its cache, or a parent cache group. The management port
+is not exposed locally - however the main content port is.
+
+* URL: [`http://localhost:8080`](http://localhost:8080)
+
+### Mid-Tier Cache
+A mid-tier cache serves content internally - within the CDN. Typically, requests are
+made from an edge-tier cache and the mid serves content from its own cache, a parent
+mid-tier cache or directly from the origin. The management port is not exposed locally -
+however the main content port is.
+
+* URL: [`http://localhost:9080`](http://localhost:9080)
+
+### Origin Server
+An origin server simply serves HTTP(S) content. The CDN-in-a-box origin server serves up
+a very simple page sporting the Traffic Control logo.
+
+* URL: [`http://localhost`](http://localhost)
+
+* URL: [`http://localhost:8081`](http://localhost:8081)
 
 The process creates containers for each component with ports exposed on the host.  The
 following should be available once the system is running:
-   
-    Traffic Portal: https://localhost    
-    Traffic Ops (go): https://localhost:6443
-    Traffic Ops (perl): https://localhost:60443
-    Postgres: `psql -h localhost -p 5432 -U postgres`
+
+	Traffic Portal: https://localhost
+	Traffic Ops (go): https://localhost:6443
+	Traffic Ops (perl): https://localhost:60443
+	Postgres: `psql -h localhost -p 5432 -U postgres`
```

Comment:
these are repeated throughout the README.md.  I think they should be summarized here and not listed with each component.

Reply:
My concern is that every component is not equal; Traffic Portal requires login credentials which ought to be documented whereas Traffic Monitor does not. It feels weird to not group the information for each component at that point. I left this in by accident - it was part of the old README that I forgot to delete.

Reply:
the intent here is for the creds to be provided by the user in the env vars.  Document the env var,  not the actual user/pass we're using as a demonstration.